tests/host_callback_test.py

```diff
@@ -141,10 +138,10 @@ def test_eval(self):
     self.assertAllClose((5. * 2.) ** 2, fun1(5.))
     hcb.barrier_wait()
     assertMultiLineStrippedEqual(self, """
-what: a * 2
-10.00
-what: y * 3
-30.00""", testing_stream.output)
+        what: a * 2
```

Comment:
I see the appeal of this new formatting, but I worry that this further increases the cost of maintaining these assertions. My workflow is to copy and paster from a failed test run directly into the assertion (after visual inspection). Now, I would have to further indent it (which is only some mouse selection and a few keystrokes, but still).

Reply:
I agree, this is an annoyance for maintenance, though I guess the existing state is pretty annoying, too.

Note that `dedent()` is invariant to indentation, so you could remove it from any of these and the tests would still pass. That said, I am also happy to revert now if you feel that would make maintenance easier.

Reply:
I do not think you need to revert, I am just pointing out that it may be a bit too much to expect that when these get revised in the future, they will stay indented. 

BTW, my IDE collapses the multi-line strings by default so these actually do not take screen space nor look ugly.